frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -446,6 +545,7 @@ class Worksheet extends React.Component {
                     blocks: Object.assign([], this.state.ws.info.blocks, { [focusIndex]: item }),
                 },
             },
+            searchExpandedBlock: searchBlocks,
```

Comment:
What is `searchExpandedBlock` used for? I don't see it used anywhere in the code.

Reply:
Good catch, it was used before the sort key change